test/goth_test.exs

```diff
@@ -110,7 +110,7 @@ defmodule GothTest do
     config = [
       name: test,
       source: {:service_account, random_service_account_credentials(), url: "http://localhost:#{bypass.port}"},
-      http_client: {Goth.HTTPClient.Hackney, []}
+      http_client: {Goth.HTTPClient.Finch, []}
```

Comment:
> `test "http client with old implementation", %{test: test} do`

please remove this test. I'm thinking that maybe doesn't make sense to keep "old" http contract after all but we don't have to make decision now.

Reply:
agreed, we can do it on v1.3 final or 1.4 or whatever. I'll deprecate the module on master.